src/main/resources/templates/email/ltft-updated/v1.0.0.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org">
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+  <title>LTFT Application Status Update</title>
+</head>
+<body>
+<h1>Email Message</h1>
+<h2>Subject</h2>
+<th:block th:fragment="subject">Your LTFT Application Status Update</th:block>
+<h2>Content</h2>
+<th:block th:fragment="content">
+  <p style="text-align:right">
+    <img style="margin-right:25px" src='https://trainee.tis.nhs.uk/nhse-logo.png' alt="NHS England logo"/>s\dfgv
+  </p>
+  <p th:replace="~{fragments/greeting/v1.0.0 :: p}">grfs\d</p>
```

Comment:
The `grfs\d` as well

Reply:
yes, they were for debugging an issue i had